### ISSUE TITLE
Add support for 'Split & Index' extractor type

### ISSUE THREAD

**Issue by CVirus** [suzuki-shunsuke/go-graylog#251]
## API Endpoints
`POST /system/inputs/{inputId}/extractors`

Sample usage would look something like:
```
resource "graylog_extractor" "x_forwarded_for" {
  input_id        = graylog_input.kube_gelf_udp.id
  title           = "x_forwarded_for"
  type            = "split_and_index"
  cursor_strategy = "copy"
  source_field    = "x-forward-for"
  target_field    = "real_ip"
  condition_type  = "none"
  condition_value = "none"
  order           = 1

  split_and_index_type_extractor_config {
    index                      = 1
    split_by                   = ","
  }
}
```

**Comment by suzuki-shunsuke** [comment]
Thank you for your feedback.
I have created a pull request https://github.com/suzuki-shunsuke/go-graylog/pull/252 .

**Comment by suzuki-shunsuke** [comment]
@CVirus 

I have released the pre release version [v11.3.0-0](https://github.com/suzuki-shunsuke/go-graylog/releases/tag/v11.3.0-0).
Please check.

If there is no problem, I'll release v11.3.0 and close this issue.

Thanks.

**Comment by CVirus** [comment]
@suzuki-shunsuke tested and everything is working perfectly.. closing issue.. Thank you so much for your time!

**Comment by suzuki-shunsuke** [comment]
Thank you for your test.
I have released [v11.3.0](https://github.com/suzuki-shunsuke/go-graylog/releases/tag/v11.3.0).

**Comment by CVirus** [comment]
@suzuki-shunsuke is the quay repo no longer maintained?
